### DISCUSSION TITLE
[Help Wanted] Questions specific to Listed

### DISCUSSION THREAD

**Discussion by MaudMan** [standardnotes/forum#2557] [General]
Hi All: 

Just a couple, hopefully simple, questions regarding Listed...

1. Is it possible to use the Listed functions (ie. publishing a note) via the Android mobile app?

2. Is there a way to include pictures/images in notes that are going to be published to Listed? I'm looking for something somewhat "frictionless" so it doesn't really get in the way of writing & creativity. 

Thanks!

**Comment by MaudMan** [comment]
I found my own answer to #1 - it's NO. :)  Still wondering about #2.

**Comment by atmoio** [comment]
Publishing via mobile is something on our radar for sure (cc @antsgar). As for #2, not yet something we're thinking about tbh, but something we'll want to tackle one day. For now you'll still have to host your images externally.

**Comment by MaudMan** [comment]
Thanks for the response Mo! Publishing via mobile will be pretty awesome. 

Do you have any recommendations for places to host images externally? Since I connected my mauder.com domain to my Listed blog, I can't host my images there anymore. Actually... since I'm still paying for hosting space for another domain I own, I could probably host some images there. But my issue is, it's still too much "friction" to upload the images I want to use in a blog post, and then link to them. :)  

-Dave

**Comment by JaspalSuri** [comment]
>Do you have any recommendations for places to host images externally?

I haven't used it and it's not affiliated with or endorsed by Standard Notes, but one open source solution that comes to mind is [Chevereto](https://chevereto.com/). 🙂 

>But my issue is, it's still too much "friction" to upload the images I want to use in a blog post, and then link to them. :)

We understand. Hopefully it's something that we can add in the future. We do plan to add support for natively uploading and viewing images within the apps, and might provide the ability to include uploaded images in published posts in a subsequent update. 🙂

**Comment by RickCogley** [comment]
At first I thought FileSafe was the answer for images for listed.to blog posts, but, obviously they are encrypted on the provider side. The paradigm of uploading to a FileSafe integration would still work though: basically select a provider, upload the image, and click a "copy insert link" button to get the appropriate markdown. 

It's easy enough to use Transmit (etc), which is a client for sftp / s3 etc, to upload into an S3 bucket (cheap to use), and it lets you easily copy the URL as well. If you're using markdown, you just need to know the `![]()` syntax. 

(These days, the javascript-y integrations where you just drag an image or paste from clipboard into a text area (like this one), and it automatically uploads to the service's CDN are so slick... Dare I say you could even just do that exact thing and repurpose the markdown that gets made here on GH?)

**Comment by fedeb95** [comment]
publishing to listed from mobile (iOS for me) would be ideal, any plan for this to happen?

**Comment by atmoio** [comment]
Yes, hopefully commencing development on this soon 🙂
